app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,136 @@
+class OrdersController < ApplicationController
+  
+  before_action :authenticate_user!
+  before_action :set_journey, only: %i(new create checkout)
+  before_action :set_order, only: %i(checkout charge)
+  before_action :instantiate_order, only: %i(create)
+  before_action :ensure_booking_is_active, only: %i(new checkout create)
+  
+  def checkout
+  end
+  
+  def new
+    @minutes = calculate_minutes
+    @order = Order.new
```

Comment:
I would like to call the method `instantiate_order`from line 98 instead of doing this. How could I call `instantiate_order`(it needs an argument in line 98 but here it does not).